src/build.d

```diff
@@ -445,7 +448,7 @@ alias runCxxUnittest = makeRule!((runCxxBuilder, runCxxRule) {
         .name("cxx-unittest")
         .description("Build the C++ unittests")
         .msg("(DMD) CXX-UNITTEST")
-        .deps([lexer, backend, cxxFrontend])
+        .deps([lexer(null), backend(null), cxxFrontend])
```

Comment:
Were does `lexer(null)` get written to? Does it overwrite the previously built lexer archive?

Reply:
No, these targets use `memoize` to create and store unique instances.

Reply:
I think the comment is about the actual binary, and valid. For example `dmdExe`'s first argument is there for this very reason.

Reply:
I see. Agreed on the binary issue, see https://github.com/dlang/dmd/pull/10930#pullrequestreview-381013711